google-cloud-tools-plugin/src/com/google/gct/idea/appengine/util/CloudSdkUtil.java

```diff
@@ -117,6 +117,10 @@ public static String toSdkHomeDirectory(String sdkExecutablePath) {
     return null;
   }
 
+  public static String getSystemCommand() {
```

Comment:
Sorry for all the chatter on this. I think its not worth the extra baggage of passing the environmentProvider around in more places just to test that method (which all the test would do is assert that our constants are written correctly). Also having a test like the snippet I pasted above feels flaky to me. So I'll just leave this test out if that sounds ok.